homeassistant/components/ecoal_boiler/http_iface.py

```diff
@@ -0,0 +1,602 @@
+"""
```

Comment:
This is device specific code. It should be move to a Python module which is published on PyPI and hen imported by the component.

Reply:
Done as https://pypi.org/project/ecoaliface/ .